### PR TITLE
Extract `get_ts_utils()` method

### PR DESCRIPTION
**Pull request by mrossinek**
Something is not yet quite working as expected because the `completion` module appears to be unhappy with the way I am trying to load things.
Hopefully @vhyrro knows what's going on here. Nonetheless, this is hopefully a decent starting point.

### CODE REVIEW

File: lua/neorg/modules/core/norg/esupports/module.lua [hunk 1]
```diff
@@ -403,8 +403,7 @@ module.public = {
                 end,
 
                 get_text_as_one = function(self, node)
-                    local ts = require("nvim-treesitter.ts_utils")
-                    return table.concat(ts.get_node_text(node, self.buf), "\n")
+                    return table.concat(ts.public.get_ts_utils().get_node_text(node, self.buf), "\n")
```
Comment (mrossinek) [inline lua/neorg/modules/core/norg/esupports/module.lua:406]:
```suggestion
                    return table.concat(ts.get_ts_utils().get_node_text(node, self.buf), "\n")
```

File: lua/neorg/modules/core/norg/esupports/module.lua [hunk 2]
```diff
@@ -950,7 +949,7 @@ module.public = {
                 else
                     vim.cmd("silent !start " .. vim.fn.fnameescape(destination))
                 end
-                return utility.ts.get_node_range(require("nvim-treesitter.ts_utils").get_node_at_cursor())
+                return utility.ts.get_node_range(utility.ts.public.get_ts_utils().get_node_at_cursor())
```
Comment (mrossinek) [inline lua/neorg/modules/core/norg/esupports/module.lua:952]:
```suggestion
                return utility.ts.get_node_range(utility.ts.get_ts_utils().get_node_at_cursor())
```

File: lua/neorg/modules/core/norg/completion/module.lua [hunk 3]
```diff
@@ -228,7 +235,7 @@ module.public = {
 					-- If the completion data has a node variable then attempt to match the current node too!
 					if completion_data.node then
 						-- Grab the treesitter utilities
-						local ts = require('nvim-treesitter.ts_utils')
+						local ts = module.private.treesitter_integration.public.get_ts_utils()
```
Comment (mrossinek) [inline lua/neorg/modules/core/norg/completion/module.lua:238]:
```suggestion
						local ts = module.private.treesitter_integration.get_ts_utils()
```

**Review by mrossinek — COMMENTED** [review]
I think I messed up a rebase here..

**Comment by vhyrro** [comment]
:purple_heart:

**Pull request merged**